althea_kernel_interface/src/exit_server_tunnel.rs

```diff
@@ -89,6 +102,64 @@ impl dyn KernelInterface {
             ],
         )?;
 
+        // Set up link local mesh ip in wg_exit as fe80 + rest of mesh ip of exit
+        if let IpAddr::V6(ip6) = exit_mesh {
+            let seg = ip6.segments();
+            let local_link = IpAddr::V6(Ipv6Addr::new(
+                0xfe80, seg[1], seg[2], seg[3], seg[4], seg[5], seg[6], seg[7],
+            ));
+            let _output = self.run_command(
+                "ip",
+                &[
+                    "address",
+                    "add",
+                    &format!("{}/64", local_link),
+                    "dev",
+                    "wg_exit",
+                ],
+            )?;
+        }
+
+        // Add iptable routes between wg_exit and eth0
+        let output = self
+            .run_command(
+                "ip6tables",
+                &[
+                    "-C", "FORWARD", "-i", "wg_exit", "-o", "eth0", "-j", "ACCEPT",
```

Comment:
this should use the external_nic parameter in rita_common network settings. See the ipv4 nat rules below

Reply:
fixed